packages/mint-components/src/components/tax-and-cash/sqm-indirect-tax-form/sqm-indirect-tax-form.feature

```diff
@@ -0,0 +1,271 @@
+    Then the radio option <option> is auto-selected
+    And the "indirectCountryCode" field <mayBe> set to the countryCode selected in Step 1
+
+    Examples:
+      | isSupported      | option        | mayBe  |
+      | is supported     | registered    | is     |
+      | is not supported | notRegistered | is not |
+
+  @minutia @ui
+  Scenario Outline: indirectTaxId field label changes depending on tax type of country selected
+    Given the radio option "registered" is selelected
+    And the "indirectCountryCode" field has <country> selected
+    And the tax type of that country is <taxType>
+    Then the label of the "indirectTaxId" field is <label>
+
+    Examples:
+      | country     | taxType       | label               |
+      | New Zealand | GST           | GST Number          |
+      | Malaysia    | SST           | SST Number          |
+      | Japan       | CT            | CT Number           |
+      | Italy       | VAT           | VAT Number          |
+      | Uganda      | Not specified | Indirect Tax Number |
+
+  @minutia @ui
+  Scenario Outline: Canada special cases
+    Given the radio option "registered" is selelected
+    And the "indirectCountryCode" field has "Canada" selected
+    Then the "province" field is displayed
+    When the "province" field has value <province>
+    Then the "indirectTaxId" field is shown and has label <label>
+    And <additionalFields> are shown
+
+    Examples:
+      | province               | label      | additionalFields |
+      | Alberta                | GST Number | n/a              |
+      | British Columbia       | HST Number | n/a              |
+      | Manitoba               | HST Number | n/a              |
+      | New Brunswick          | HST Number | n/a              |
+      | Newfoundland           | HST Number | n/a              |
+      | North West Territories | HST Number | n/a              |
+      | Nova Scotia            | HST Number | n/a              |
+      | Nunavut                | HST Number | n/a              |
+      | Ontario                | HST Number | n/a              |
+      | Prince Edward Island   | HST Number | n/a              |
+      | Quebec                 | GST Number | registeredForQST |
+      | Saskatchewan           | HST Number | n/a              |
+      | Yukon                  | GST Number | n/a              |
+
+  @minutia @ui
+  Scenario: Selecting the "registeredForQST" checkbox shows QST Number input
+    Given the radio option "registered" is selelected
+    And the "indirectCountryCode" field has "Canada" selected
+    And the "province" field has value "Quebec"
+    Then the "registeredForQST" checkbox with label "I am registered for QST Tax" will appear
+    When the "registeredForQST" checkbox is selected
+    Then the "qstNumber" field is shown
+    And the "qstNumber" field has label "QST Number"
+
+  @minutia @ui
+  Scenario Outline: Participant selects other country that has tax handling in step 1
```

Comment:
flagging need for update

Reply:
Updated to explain the regular indirect tax country selection case